src/lib/docs/Antelope/Struct.md

```diff
@@ -31,7 +30,7 @@ The `Struct` type in the Antelope library represents one entry in the `structs`
 }
 ```
 
-This on-chain representation of a `Struct` allows the Antelope blockchain to encode and decode data of this type for use in smart contracts. In a similar way, the `Struct` found in the Antelope library also allows the [Serializer](/docs/antelope/serializer) to encode and decode this data within Javascript applications. Many of these `Struct` objects exist as core types offered by the library, such as [Asset](/docs/antelope/asset), [Name](/docs/antelope/name), and [PermissionLevel](/docs/antelope/permission-level).
+This on-chain representation of a `Struct` allows the Antelope blockchain to encode and decode data of this type for use in smart contracts. In a similar way, the `Struct` found in the Antelope library also allows the [Serializer](/docs/antelope/serializer) to encode and decode this data within Javascript applications. Many of these `Struct` objects exist as core types offered by the library, such as [Asset](/docs/antelope/asset), [Name](/docs/antelope/name), and [Permission Level](/docs/antelope/permission-level).
```

Comment:
In this case, I think `PermissionLevel` is the name of the thing and can't be split.

Reply:
fixed